### ISSUE TITLE
Dashboard gives "403 Forbidden" alerts when following Astarte in 5 minutes

### ISSUE THREAD

**Issue by davidebriani** [astarte-platform/astarte#476]
The guide `Astarte in 5 minutes` > `Grab your tea` suggests to login in the Dashboard with a JWT token generated with:
```
astartectl utils gen-jwt realm-management -k test_private.pem
```
This is not sufficient to interact with all APIs the Dashboard support, Devices for instance.
The suggested command to generate the JWT token should be:
```
astartectl utils gen-jwt all-realm-apis -k test_private.pem
```

**Comment by Aetherus** [comment]
When monitoring a device on the dashboard, the websocket connection request to the appengine-api's `/socket/websocket` gives `403 Forbidden` even when the JWT token is for all realm apis.

**Comment by Pavinati** [comment]
@Aetherus when generating tokens with `astartectl` the default expiry is 5minutes, make sure the token is still valid before connecting to the socket.
If you need longer expiration dates you can use the `-e` flag and set a custom value (in seconds)

If the token was valid, you can inspect its claims to make sure you have both `JOIN` and `WATCH` permission on the Astarte channels
```
  "a_ch": [
    "JOIN::.*",
    "WATCH::.*"
  ],
``` 
If you need a quick inspection tool you can use https://jwt.io/

**Comment by rbino** [comment]
Closed by #537
